packages/boot/test/unit/booters/repository.booter.unit.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect, TestSandbox, sinon} from '@loopback/testlab';
+import {Application} from '@loopback/core';
+import {RepositoryMixin} from '@loopback/repository';
+import {RepositoryBooter, RepositoryDefaults} from '../../../index';
+import {resolve} from 'path';
+
+describe('repository booter unit tests', () => {
+  const SANDBOX_PATH = resolve(__dirname, '../../../.sandbox');
+  const sandbox = new TestSandbox(SANDBOX_PATH);
+
+  const REPOSITORIES_PREFIX = 'repositories';
+  const REPOSITORIES_TAG = 'repository';
+
+  class RepoApp extends RepositoryMixin(Application) {}
+
+  let app: RepoApp;
+  let stub: sinon.SinonStub;
+
+  beforeEach(() => sandbox.reset());
```

Comment:
`await` seems missing for `sandbox.reset()`

Reply:
Mocha automatically waits for promises. 

Source: 
1. https://stackoverflow.com/a/24862303
2. https://mochajs.org/#working-with-promises

Reply:
On second thought, you're right since the lambda function being passed is not returning a promise, the hook will not wait for anything. I've fixed the code.

Reply:
your original code is correct 👍 
I just tested `await aPromise`, it still waits until the promise is resolved, see comment in https://github.com/strongloop/loopback-next/pull/1043#issuecomment-368090125